common-rendering/src/components/keyEvents.stories.tsx

```diff
@@ -68,26 +59,43 @@ const events: KeyEvent[] = [
 	},
 ];
 
-const Default = (args: { theme: Theme }) => (
-	<KeyEvents keyEvents={events} {...args} />
+const keyEventComp = (dark: boolean, theme: Theme, title: string) => (
```

Comment:
Don't React components have to start with a capital letter? Or is that just in my head?

Reply:
They do, apparently